flyteidl/protos/flyteidl/core/literals.proto

```diff
@@ -102,6 +102,10 @@ message Literal {
 
         // A map of strings to literals.
         LiteralMap map = 3;
+
+        // Offloaded literal metadata
+        // When you deserialize the offloaded metadata, it would be of Literal and its type would be defined by LiteralType stored in offloaded_metadata.
+        LiteralOffloadedMetadata offloaded_metadata = 6;
```

Comment:
Since we have a published flyteidl version that contains those 2 fields, can we use 8 and [reserve fields 6 and 7](https://protobuf.dev/programming-guides/dos-donts/#reserve-tag-numbers)?

Reply:
Done. thank @eapolinario